api/backend/ecs/resource_manager.go

```diff
@@ -57,7 +57,7 @@ func (r *ECSResourceManager) GetProviders(environmentID string) ([]*resource.Res
 func (r *ECSResourceManager) getResourceProvider(ecsEnvironmentID id.ECSEnvironmentID, instance *ecs.ContainerInstance) (*resource.ResourceProvider, bool) {
 	instanceID := pstring(instance.Ec2InstanceId)
 	if pstring(instance.Status) != "ACTIVE" {
-		r.logger.Warnf("Instance %d is not in the 'ACTIVE' state", instanceID)
+		r.logger.Warnf("Instance %s is not in the 'ACTIVE' state", instanceID)
```

Comment:
Is this just a linter fix? What's the intention of this change?

Reply:
I was getting errors when these things mismatched. I'm on Go 1.12; are you? Maybe you could try playing around with it?

```
package main

import "fmt"

func main() {
    fmt.Printf("percent-s and %s\n", "string")
    fmt.Printf("percent-s and %s\n", 2)
    fmt.Printf("percent-d and %d\n", "string")
    fmt.Printf("percent-d and %d\n", 2)
}
```

Reply:
yep I get compile errors as well. Did Go 1.12 change its' behavior with `%d`? I guess so?

Reply:
Yeah, that's what it seems like to me.